src/pivot.coffee

```diff
@@ -464,7 +467,7 @@ callWithJQuery ($) ->
                             agg = agg[aggIdx or 0]
                         return agg.value()
 
-                    keys.sort (a,b) => naturalSort(_getVal(a), _getVal(b)) * (if isDesc then -1 else 1)
+                    keys.sort (a,b) => getSort(@sorters, null)(_getVal(a), _getVal(b)) * (if isDesc then -1 else 1)
```

Comment:
Why is `_convertBlankToNull` not needed here?

Reply:
This is sorting based on aggregated values, NOT based on attribute values. `_convertEmptyToNull` is only needed when sorting attribute values. When sorting aggregated values, we need the actual attribute value to look up the corresponding aggregated value.